runtime/jcl/common/system.c

```diff
@@ -477,10 +477,18 @@ jobject getPropertyList(JNIEnv *env)
 		javaVM->directByteBufferMemoryMax = heapSize - (heapSize / 8);
 	}
 #endif /* defined(OPENJ9_BUILD) && JAVA_SPEC_VERSION == 8 */
-	if ((UDATA) -1 != javaVM->directByteBufferMemoryMax) {
+#if !defined(OPENJ9_BUILD)
+	/* Don't set a default value for IBM Java 8 .*/
```

Comment:
nit: the space should be after the period (not before)

Reply:
Oops, fixed.